tests/jlm/hls/backend/rvsdg2rhls/DeadNodeEliminationTests.cpp

```diff
@@ -15,10 +15,10 @@ TestDeadLoopNode()
   using namespace jlm::hls;
 
   // Arrange
-  jlm::tests::valuetype valueType;
+  auto valueType = std::make_shared<jlm::tests::valuetype>();
```

Comment:
Could we get a `Create()` method on the `jlm::tests::valuetype`?

Reply:
done

Reply:
Could we make use of the newly introduced `Create()` method here?

Reply:
sorry, systematically forgot this search/replace pattern... all fixed now